lib/APPSensorFusion/src/DrivingState.cpp

```diff
@@ -57,28 +58,57 @@
  * Prototypes
  *****************************************************************************/
 
+#ifdef DEBUG_ALGORITHM
+static void logCsvDataTitles(uint8_t length);
+static void logCsvDataTimestamp();
+static void logCsvData(const uint16_t* lineSensorValues, uint8_t length, int16_t pos, int16_t pos3, bool isPos3Valid);
+void        logCsvDataTrackStatus(DrivingState::TrackStatus trackStatus);
+static void logCsvDataSpeed(int16_t leftSpeed, int16_t rightSpeed);
+#endif /* DEBUG_ALGORITHM */
+
 /******************************************************************************
  * Local Variables
  *****************************************************************************/
 
+#ifdef DEBUG_ALGORITHM
```

Comment:
Is it intended to include all there DEBUG blocks?

Reply:
removed